docs/walkthroughs/02-adding-event-handlers.md

```diff
@@ -75,7 +75,7 @@ const App = () => {
             // Prevent the ampersand character from being inserted.
             event.preventDefault()
             // Execute a command to insert text when the event occurs.
-            editor.exec({ type: 'insert_text', text: 'and' })
+            Editor.insertText(editor, 'and')
```

Comment:
where is `Editor` coming from?

Reply:
https://github.com/ianstormtaylor/slate/blob/8cd9b793528eb76a4a76940e022854cf33419bc4/site/examples/richtext.js#L4